pkg/controller/lifecycle/actuator.go

```diff
@@ -200,7 +200,8 @@ func (a *Actuator) Reconcile(ctx context.Context, logger logr.Logger, ex *extens
 	}
 
 	// Reconcile managedresource and secret for shoot.
-	if err := managedresources.CreateForShoot(ctx, a.client, namespace, consts.ManagedResourceName, false, secretData); err != nil {
+	origin := "gardener"
```

Comment:
Have you checked https://github.com/gardener/gardener/blob/master/docs/concepts/resource-manager.md#origin
for value of origin.